package.json

```diff
@@ -58,14 +58,14 @@
   "scripts": {
     "hooks:install": "husky install",
     "hooks:uninstall": "husky uninstall",
-    "build": "turbo run build",
+    "build": "turbo run build --no-daemon",
```

Comment:
`--no-daemon` is necessary to prevent turbo to run as a standalone process. Known to cause issues on some host environment... like mine.